### ISSUE TITLE
Info Event occurred on removed layers

### ISSUE THREAD

**Issue by Sebastian1989101** [pauldendulk/Mapsui#185]
Near identical to issue #182. If I add a layer to my app and remove it at runtime, the Info event still occurs in the position where one icon from the removed layer was. 

Video about the issue:
https://secure.kruse-familie.eu/bug-mapsui-info-event-on-removed-layer.avi

**Comment by pauldendulk** [comment]
The Info event is invoked on the list of InfoLayers, see code below. Your problem could be solved by also removing it from the InfoLayer list.

```
        public void InvokeInfo(Point screenPosition, ISymbolCache symbolCache)
        {
            if (Info == null) return;
            var eventArgs = InfoHelper.GetInfoEventArgs(Viewport, screenPosition, InfoLayers, symbolCache);
            if (eventArgs != null) Info?.Invoke(this, eventArgs);
        }
```
This is double administration is not that great. Alternatives:
- Adding an Info property to a layer and use that instead of the InfoLayers list.
- Check if InfoLayer is still in main list. This would leave dangling InfoLayers, not great.

**Comment by pauldendulk** [comment]
Lets go for option 3

**Comment by pauldendulk** [comment]
Fixed in 1.0.2
